mlir/test/Integration/Dialect/Vector/CPU/test-rewrite-narrow-types.mlir

```diff
@@ -164,6 +164,13 @@ func.func @fext(%a: vector<5xi8>) {
   return
 }
 
+func.func @fcst_maskedload(%A: memref<?xi4>, %passthru: vector<6xi4>) -> vector<6xi4> {
+  %c0 = arith.constant 0: index
+  %mask = vector.constant_mask [3] : vector<6xi1>
+  %1 = vector.maskedload %A[%c0], %mask, %passthru :
+    memref<?xi4>, vector<6xi1>, vector<6xi4> into vector<6xi4>
+  return %1 : vector<6xi4>
+}
```

Comment:
I just noticed that this is not tested because the patterns are not populated..

https://github.com/llvm/llvm-project/blob/a0bd6361d4840f159e48c7359807ae279d006fb0/mlir/lib/Dialect/Vector/TransformOps/VectorTransformOps.cpp#L162-L165

To test it, we need to set up a type converter and populate those patterns.